scripts/prepublish.mjs

```diff
@@ -93,13 +93,17 @@ const downloadMicrobitHex = async () => {
     const absoluteGeneratedDir = path.join(basePath, relativeGeneratedDir);
     fs.mkdirSync(absoluteGeneratedDir, {recursive: true});
     const absoluteGeneratedFile = path.join(basePath, relativeGeneratedFile);
+    const requirePath = `./${path
+        .relative(relativeGeneratedDir, relativeHexFile)
+        .split(path.win32.sep)
+        .join(path.posix.sep)}`;
```

Comment:
`.replaceAll(path.win32.sep, path.posix.sep)` would have been cleaner in my opinion, but that was introduced in Node 15. Pretty sure part of our CI is using Node 14.

Reply:
Or, maybe just `.replace(/[\\/]g/, '/')` 😸

Reply:
I just noticed that @xxtlichao did exactly that in #9033!